client/src/app/shared/components/toasts.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, TemplateRef } from '@angular/core';
+import { ToastService } from '../services/toast.service';
+
+@Component({
+  selector: 'app-toasts',
+  template: `
```

Comment:
https://angular.io/guide/styleguide#extract-templates-and-styles-to-their-own-files